legacy/firmware/udp.c

```diff
@@ -63,12 +64,12 @@ void usbSleep(uint32_t millis) {
 #endif
 }
 
-void usbPoll(void) { usbSleep(0); }
+void usbPoll(void) { waitAndProcessUSBRequests(0); }
 
 char usbTiny(char set) {
   char old = tiny;
   tiny = set;
   return old;
 }
 
-void usbFlush(uint32_t millis) { usbSleep(millis); }
+void usbFlush(uint32_t millis) { usleep(millis * 1000); }
```

Comment:
This doesn't flush anymore though. We actually need to call "process USB requests" to do that. We might want to separate "write out pending" from "read incoming" for the purpose of this function.

Reply:
Fixed by writing out oustanding packets.